beacon-chain/chaintest/backend/helpers.go

```diff
@@ -37,6 +47,50 @@ func generateSimulatedBlock(
 			Exits:             []*pb.Exit{},
 		},
 	}
+	if simulatedDeposit != nil {
+		depositInput := &pb.DepositInput{
+			Pubkey:                      []byte(simulatedDeposit.Pubkey),
+			WithdrawalCredentialsHash32: []byte{},
+			ProofOfPossession:           []byte{},
+			RandaoCommitmentHash32:      depositRandaoCommit[:],
+			CustodyCommitmentHash32:     []byte{},
+		}
+		wBuf := new(bytes.Buffer)
+		if err := ssz.Encode(wBuf, depositInput); err != nil {
+			return nil, [32]byte{}, fmt.Errorf("failed to encode deposit input: %v", err)
+		}
+		encodedInput := wBuf.Bytes()
+		data := []byte{}
+
+		// We set a deposit value of 1000.
```

Comment:
Instead of this whole chunk, why not use `EncodeDepositdata` ? it does the same thing